litmus-portal/frontend/src/components/WelcomeModal/Stepper.tsx

```diff
@@ -235,7 +250,7 @@ function CStepper() {
               </div>
             }
             // pass here corresponding name of user
-            setName="Name"
+            setName={info.name}
```

Comment:
Why is name, email and password set to `info.name` ?

Reply:
It shouldn't be hardcoded right? These are not the fields that I am setting to info.data, it's the text that is displayed on the modal, "Welcome `name`'